pkgs/development/tools/misc/texinfo/common.nix

```diff
@@ -14,9 +14,19 @@
 
 let
   crossBuildTools = stdenv.hostPlatform != stdenv.buildPlatform;
-in
 
-with lib;
+  inherit (lib)
+    getDev
+    getLib
+    licenses
+    maintainers
```

Comment:
Side note: I'm on the fence about this. `licenses` and `maintainers` are used only locally, typically in one place per file, and I wouldn't mind to have them `with`'ed there. Having a `let ... in` per `meta` seems a bit much, but may not be wrong. I think this change okay-ish, and people can always clean it up if they touch the code and feel hindered by doing it that way.

Reply:
This is an artifact of my (maybe bone-headed) tool-driven refactor.

It:

1. Pulls out names if they're ever in a top-level `with`
2. Removes intermediate `with` statements that were identical to the top-level `with` (that's why the lower `with` is removed!)
3. Unifies use of that attrset with `inherit (a) thing`.

This is the number 1 thing that I've gotten feedback about in PRs like these, and I'm both tired of getting the feedback and tired of the feeling of leaving something undone when I don't apply these principles.

What would you recommend I do?